submissions/webdevtoday/task_js_dom/js/Controller.js

```diff
@@ -0,0 +1,27 @@
+import {Render} from './Render.js';
+
+export class Controller {
+    constructor(root) {
+        this.root = root;
+        this.render = new Render( this.root );
+    }
+
+    get(controllerName) {
+        if (!controllerName) {
+            this.render.render( `<h1>404 Page Not Found</h1>` );
```

Comment:
Maybe it's better to rename this variable to another one because `this.render.render` is a confusing a little bit.
```suggestion
            this.renderInstance.render( `<h1>404 Page Not Found</h1>` );
```

Reply:
That's true! I'll change it

Reply:
I did it